ui/src/components/dialog/QasDialog.vue

```diff
@@ -172,6 +172,20 @@ export default {
 
     isInfoDialog () {
       return !this.hasOk && !this.hasCancel
+    },
+
+    formattedActionsProps () {
+      const { useFullWidth, useEqualWidth } = this.actionsProps
+
+      if (useFullWidth || useEqualWidth) {
```

Comment:
Por que precisa fazer esse `if` se abaixo você já faz o spread do `...this.actionsProps` logo após as pré definições do `useFullWidth` e `useEqualWidth`? 🤔

Reply:
é pq o que estava acontecendo é que  o spread sobrescrevia o valor da prop q eu estava passando (useFullWidth) mas o useEqualWidth ainda era mantido como true, e o actions dava preferencia ao useEqualWidth. Ai dessa forma, se estou passando alguma dessas props, ele ignora os valores defaults do componente. Outra forma seria
```js
formattedActionsProps () {
   const { useFullWidth, useEqualWidth } = this.actionsProps
   
   return {
      useFullWidth: useFullWidth || this.hasSingleAction,
      useEqualWidth: useEqualWidth || this.hasAllActions,
      ...this.actionsProps
   }
}
```